tracer/src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -362,6 +362,7 @@ private static void InitRemoteConfigurationManagement(Tracer tracer)
 
                         var configurationManager = RemoteConfigurationManager.Create(discoveryService, rcmApi, rcmSettings, serviceName);
                         configurationManager.RegisterProduct(SharedRemoteConfiguration.FeaturesProduct);
+                        configurationManager.RegisterProduct(SharedRemoteConfiguration.AsmDDProduct); // This should be activated by Security only when Asm is active, but there is no visibility right now
```

Comment:
I'm surely missing smth, but why can't this be registered in the `EnableWaf` function of the `Security` class?

Reply:
Because configurationManager is not available there.

Reply:
ha right... yes I think it should be made available at some point to the Security class... I remember discussions as to make it part of `TracerManager`